Stripe/PublicHeaders/STPPaymentIntentShippingDetailsAddressParams.h

```diff
@@ -0,0 +1,64 @@
+//
+//  STPPaymentIntentShippingDetailsAddressParams.h
+//  Stripe
+//
+//  Created by Yuki Tokuhiro on 4/27/20.
+//  Copyright © 2020 Stripe, Inc. All rights reserved.
+//
+
+#import <Foundation/Foundation.h>
+
+#import "STPFormEncodable.h"
+
+NS_ASSUME_NONNULL_BEGIN
+
+/**
+Shipping address for a PaymentIntent's shipping details.
+
+@see https://stripe.com/docs/api/payment_intents/confirm#confirm_payment_intent-shipping-address
+*/
+@interface STPPaymentIntentShippingDetailsAddressParams : NSObject <NSCopying, STPFormEncodable>
```

Comment:
I don't think it's needed for this patch, but we should have a plan to rectify this class with the existing `STPAddress` and other things like that